### PR TITLE
HPCC-13877 Add a bundle to create record definitions from MySQL

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

### CODE REVIEW

**Comment by ghalliday** [comment]
@rpastrana FYI
@richardkchapman please review.

File: MySqlImport/MySqlImport.ecl [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        DATA fullkind;
+        unsigned maxlen;
+    END;
+    
+    tableRecord := RECORD
+        string table_name;
+        string comment;
+    END;
+    
+    //dataset(tableRecord) myTables := EMBED(mySql : database('orderella'),user('orderella'),database('orderella'))//,server('127.0.0.1'), port('3306')
```
Comment (richardkchapman) [inline MySqlImport/MySqlImport.ecl]: Should this be deleted?

File: MySqlImport/MySqlImport.ecl [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        EXPORT Name := 'MySqlImport';
+        EXPORT Description := 'Create record and dataset definitions from MySQL tables';
+        EXPORT Authors := ['Gavin Halliday'];
+        EXPORT License := 'http://www.apache.org/licenses/LICENSE-2.0';
+        EXPORT Copyright := 'Copyright (C) 2015 HPCC Systems';
+        EXPORT DependsOn := [];
+        EXPORT Version := '1.0.0';
+    END;
+
+    string database := '' : stored('database');
```
Comment (richardkchapman) [inline MySqlImport/MySqlImport.ecl]: Not sure you want to be using stored here do you?

Comment (richardkchapman) [inline MySqlImport/MySqlImport.ecl]: (as opposed to passing in parameters)

File: MySqlImport/MySqlImport.ecl [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        string table_name;
+        string comment;
+    END;
+    
+    //dataset(tableRecord) myTables := EMBED(mySql : database('orderella'),user('orderella'),database('orderella'))//,server('127.0.0.1'), port('3306')
+    dataset(tableRecord) myTables() := EMBED(mySql : database(database),user(user),password(password))
+        SELECT table_name, table_comment FROM information_schema.tables WHERE table_type='BASE TABLE' AND table_schema='orderella';
+    ENDEMBED;
+    
+    dataset(columnRecord) getColumns(string table) := EMBED(mySql : database('orderella'),user('orderella'),database('orderella'))
```
Comment (richardkchapman) [inline MySqlImport/MySqlImport.ecl]: This should be using the stored variables (or better, module parameters) rather than constants.

**Comment by richardkchapman** [comment]
@ghalliday back to you

**Comment by ghalliday** [comment]
@richardkchapman thanks.  Cleaned up and repushed.

**Pull request merged**